samples/trumpify/trump.py

```diff
@@ -0,0 +1,83 @@
+import os
+import sys
+import time
+from threading import Thread
+
+from twitter import *
+
+sys.path.append(os.path.abspath('../../'))
+
+from daemo.client import Client
+
+CREDENTIALS_FILE = 'credentials.json'
+
+PROJECT_ID = -1
+
+TW_CONSUMER_KEY = ''
+TW_CONSUMER_SECRET = ''
+TW_ACCESS_TOKEN = ''
+TW_ACCESS_TOKEN_SECRET = ''
```

Comment:
Is this the common structure? As opposed to having these keys in the code, or be environment variables? I'm thinking that it might be difficult for a developer to push all this to Heroku, for instance, since they'd have to check their credentials file into git (yikes!). Whereas if it's in the code, they can use the environment variables.

+ @dmorina